crates/bytecode/src/utils.rs

```diff
@@ -19,3 +19,135 @@ pub unsafe fn read_i16(ptr: *const u8) -> i16 {
 pub unsafe fn read_u16(ptr: *const u8) -> u16 {
     u16::from_be_bytes(unsafe { ptr.cast::<[u8; 2]>().read() })
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_read_u16_big_endian() {
```

Comment:
For example, this one is the only needed test for `read_u16`.

`test_read_u16_all_ones` is not a valid test in this context.